src/languages/shell.rs

```diff
@@ -42,6 +48,7 @@ impl core::fmt::Display for Shell {
         match self {
             Self::Shfmt => write!(f, "shfmt"),
             Self::Beautysh => write!(f, "beautysh"),
+            Self::FishIndent => write!(f, "fish_indent"),
```

Comment:
I'm unsure I understand the code, but 

I would expect fish_ident to be used only with fish  `markdown` identifier

Fish shell is not POSIX, calling it on bash, sh, or zsh may produce invalid code

Reply:
But I'm unsure there is a standardized fish identifier

https://github.com/github-linguist/linguist/blob/master/lib%2Flinguist%2Flanguages.yml

Reply:
It's used pretty often anyway

https://github.com/search?q=%22%60%60%60fish%22+language%3Amd&type=code

Reply:
Aaah, I am not a fish user so hadn't really looked into it, just thought it was some minor differences 😅

Reply:
I would say it may happen.

Reply:
Thanks